Mathlib/Data/Array/MinMax.lean

```diff
@@ -0,0 +1,48 @@
+/-
+Copyright (c) 2023 Thomas Murrills. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Thomas Murrills
+-/
+
+/-! # Extremizing functions on Arrays
+
+This file supplements functions like `min` and `max` on `Array`s.
+
+We introduce functions like `argmin`, which collects the elements of an array that are minimal
+under some given function.
+
+We try to use `Ord` instances where possible to stay consistent with functions like `Array.min`,
+but do also provide functions for e.g. `Preorder`s.
```

Comment:
I think this is talking about some future version of this file? :-)

Reply:
Oops, you're absolutely right! Fixed in [1df5446](https://github.com/leanprover-community/mathlib4/pull/2323/commits/1df5446d2f3e0d44612346df3734d6970b5413ff).